tools/shoestring/Jenkinsfile

```diff
@@ -10,4 +10,6 @@ defaultCiPipeline {
 
 	codeCoverageTool = 'coverage'
 	minimumCodeCoverage = 95
+
+	discordWebHookUrlId = 'CLIENT_DISCORD_WEB_HOOK_URL_ID'
```

Comment:
client or infra?

Reply:
hmmm...good question.  I wasnt sure either. I was thinking Infra as CI/CD and nodes
@0x6861746366574 your thoughts?